video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
Actually another way you could do it is like this:

```glsl
    MPGL_CAP_SSBO               = (1 << 21),    // GL_ARB_shader_storage_buffer_object
    MPGL_CAP_COMPUTE            = (1 << 22),    // GL_ARB_compute_shader
    MPGL_CAP_2D_ARRAY           = (1 << 23),    // GL_ARB_array_of_arrays
```

And then we could have something like “#define CAP_COMPUTE (MPGL_CAP_COMPURE | MPGL_CAP_2D_ARRAY)` inside video.c which we can use to simplify our repeated checks for multiple features? Hard to call.